spec/books_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../classes/book'
+
+describe Book do
+  before :each do
+    @book = Book.new 'The fighter', 'Good', '2021-07-02'
+  end
+
+  context 'when testing the Book class' do
+    it 'should return true if object is an instance of the Book class' do
+      expect(@book).to be_an_instance_of Book
+    end
+
+    it 'should returning false while acquiring the cover_state property' do
+      cover_state = @book.cover_state == 'Bad'
+      expect(cover_state).to be false
+    end
+
+    it 'should return false when calling the can_be_archived? method' do
+      expect(@book.can_be_archived?).to be false
+    end
+  end
+end
```

Comment:
Tests in this file are failing. Kindly fix these errors.
![image](https://user-images.githubusercontent.com/33268927/193256174-4abe8eea-6863-43fd-8334-4ebcd0b667b5.png)